src/react-components/create-object-dialog.js

```diff
@@ -23,7 +23,7 @@ const attributionHostnames = {
 
 const isMobile = AFRAME.utils.device.isMobile() || AFRAME.utils.device.isMobileVR();
 const instructions = "Paste a URL to an image, video, model, scene, or upload.";
-const desktopTips = `Tip: You can paste URLs directly into ${messages["app-name"]} with Ctrl+V`;
+const desktopTips = `Tip: You can paste URLs directly into ${getMessages()["app-name"]} with Ctrl+V`;
```

Comment:
Oops, this is one of several remaining places where we should be using `getMessages()` for the whole text. Can be addressed later